CMakeLists.txt

```diff
@@ -49,10 +49,10 @@ list (APPEND camp_headers ${PROJECT_BINARY_DIR}/include/camp/config.hpp)
 # backends
 set (camp_depends)
 
-set (camp_backends target_openmp openmp cuda hip sycl)
+set (camp_backends openmp cuda blt::hip sycl)
 # NOTE: camp itself doesn't require a hip or cuda compiler,
 #       but some of its features are activated by having a device compiler
-set (camp_runtime_backends cuda hip)
+set (camp_runtime_backends cuda blt::hip)
```

Comment:
Should this be blt::hip_runtime instead?

Reply:
It should not, because the code adds "_runtime" to the end if the entry is in the runtimes list.

Reply:
d'oh! I didn't look further down in the file. sorry about that.